kubernetes/annotations_and_labels.md

```diff
@@ -8,6 +8,9 @@ This page defines common annotations and labels we set in Kubernetes objects.
   location. For now this should be in our `giantswarm/giantswarm` repository in
   which we crosslink all pages to create a reasonable documentation of all kinds
   of Kubernetes resources, their functionality and relationships.
```

Comment:
It should be specified where we set those annotations/labels. See https://github.com/giantswarm/fmt/blob/31541cd695647a0461069f0337f4fd3d3485e5c6/kubernetes/annotations_and_labels.md#tenant-cluster